app/classes/Transvision/Utils.php

```diff
@@ -233,13 +233,12 @@ public static function getRepoStrings($locale, $repository)
 
         $file = TMX . "{$locale}/cache_{$locale}_{$repository}.php";
 
-        if (! $tmx = Cache::getKey($file)) {
-            if (is_file($file)) {
-                include $file;
-                Cache::setKey($file, $tmx);
-            }
+        if (! is_file($file)) {
+            return [];
         }
 
+        include $file;
+
         return $tmx !== false ? $tmx : [];
```

Comment:
Yes. Transvision's log is pretty clean right now, so errors should be highly visible.